server_channel.go

```diff
@@ -91,6 +91,13 @@ func (cs *channelServer) run() {
 	elapsed = time.Since(start)
 	log.Println("Loaded and parsed reactor data in", elapsed)
 
+	start = time.Now()
```

Comment:
This could be moved into the `Initialise` method of the server struct in the channel package and you wouldn't then need to export the populate function.